tutorials/explanations/ai-explanations-tabular.ipynb

```diff
@@ -793,7 +793,9 @@
         "\n",
```

Comment:
<p>is the version upgrade required?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ml-on-gcp/pull/90/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='56'/>

Reply:
Yeah, it'll throw an error if you don't use `3.7`